testcases/misc_tests/cca_export_import_test.c

```diff
@@ -349,6 +349,62 @@ static CK_RV import_ecc_publ_key(CK_SESSION_HANDLE session,
     return rc;
 }
 
+static CK_RV import_ibm_dilithium_priv_key(CK_SESSION_HANDLE session,
+                                           const char *label,
+                                           CK_BYTE *ccatoken, CK_ULONG tokenlen,
+                                           CK_OBJECT_HANDLE *handle)
+{
+    CK_RV rc;
+    CK_OBJECT_CLASS keyClass = CKO_PRIVATE_KEY;
+    CK_KEY_TYPE keyType = CKK_IBM_DILITHIUM;
+    CK_BBOOL true = TRUE;
+    CK_BBOOL false = FALSE;
+    CK_ATTRIBUTE template[] = {
+        {CKA_CLASS, &keyClass, sizeof(keyClass)},
+        {CKA_KEY_TYPE, &keyType, sizeof(keyType)},
+        {CKA_LABEL, (char *) label, strlen(label) + 1},
+        {CKA_SIGN, &true, sizeof(true)},
+        {CKA_TOKEN, &false, sizeof(false)},
+        {CKA_IBM_OPAQUE, ccatoken, tokenlen}
+    };
+    CK_ULONG nattr = sizeof(template)/sizeof(CK_ATTRIBUTE);
```

Comment:
spaces around the / ?

Reply:
Yea, fixed many other such places, too....